src/test/moment/locale.js

```diff
@@ -504,6 +504,18 @@ test('when in strict mode with inexact parsing, treat periods in min-weekdays li
     assert.equal(moment('thurs', 'dd', true).format('dddd'), 'Thursday');
 });
 
+test('when weekdays is passed \'true\' as argument, days have to be ordered based on locale first day of the week', function (assert) {
+    var deLocaleData = moment();
```

Comment:
nit/question: any reason we can't use `const` here?

Reply:
I think the rest of this library is written in ES5, but `const` was a ES2015 addition:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const